src/gobimportclient/reader/database.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy.sql import text
+
+
+def read_from_database(connection, table):
+    """Reads from the database
+
+    The SQLAlchemy library is used to connect to the data source for databases
+
+    :return: a list of data
+    """
+    query = text("SELECT * FROM MEETBOUT")
+    result = connection.execute(query).fetchall()
```

Comment:
MEETBOUT should be a variable so that any table can be read

Reply:
Fixed, table is now selected based on variable and uses a db schema if provided